content_schemas/formats/fields_of_operation.jsonnet

```diff
@@ -0,0 +1,10 @@
+(import "shared/default_format.jsonnet") + {
+  document_type: [
+    "fields_of_operation"
+  ],
+  links: (import "shared/base_links.jsonnet") + {
+    fields_of_operation: {
+        description: "Link to an individual field of operation"
+    },
+  },
+}
```

Comment:
Nitpick: newline missing here

Reply:
curse you VS code! thanks, good spot, will turn on the feature that adds a new line 🙈